components/brave_wallet_ui/stories/locale.ts

```diff
@@ -1364,5 +1364,6 @@ provideStrings({
   braveWalletVisitDapp: 'Visit $1',
   braveWalletFilters: 'Filters',
   braveWalletClearFilters: 'Clear filters',
-  braveWalletShowMore: 'Show more'
+  braveWalletShowMore: 'Show more',
+  braveWalletDetails: 'Details'
```

Comment:
Thanks for adding a new generic "details" locale, but can we remove the old locales that also use the string "Details"? i.e: `braveWalletAllowAddNetworkDetailsPanelTitle`